### DISCUSSION TITLE
Quarkus 2.8.1 release date

### DISCUSSION THREAD

**Discussion by RuggeroDAlo** [quarkusio/quarkus#24918] [Q&A]
Has the release date for quarkus 2.8.1 been announced?
I'm especially waiting for the official release of https://github.com/quarkusio/quarkus/issues/24626 .

I don't want to sound ungrateful since 2.8.0 was just released, thanks for all the hard work :smile:

**Comment by gsmet** [comment]
My plan is to release it next week, probably on Tuesday.
